pkg/analysis/conditions/analyzer.go

```diff
@@ -113,20 +121,20 @@ func (a *analyzer) checkField(pass *analysis.Pass, index int, field *ast.Field,
 func checkFieldMarkers(pass *analysis.Pass, field *ast.Field, fieldMarkers markers.MarkerSet) {
 	missingMarkers := []string{}
 
-	if !fieldMarkers.Has(listTypeMap) {
-		missingMarkers = append(missingMarkers, listTypeMap)
+	if !fieldMarkers.HasWithExpressions(listType, map[string]string{"": listTypeExpressionValue}) {
```

Comment:
This got me thinking, what would we do about conflicting markers 🤔 If someone has `listType` with two different values 🤔

Reply:
This is a good question. I'm going to spend a bit more time thinking about how the `MarkerSet` is used and the current behaviors of the existing linters.

Reply:
I guess we probably want to create a separate linter that will check for well known markers with conflicting values, not something we necessarily need to solve in this PR